### PR TITLE
templates with questions

### PR DESCRIPTION
**Pull request by oleg08**
Did the ability to add and delete questions template

### CODE REVIEW

File: app/controllers/admin/questionstemplates_controller.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+class Admin::QuestionstemplatesController < ApplicationController
```
Comment (Kmalynss) [inline app/controllers/admin/questionstemplates_controller.rb]: we don't need separate controller for this model

File: app/controllers/admin/questions_controller.rb [hunk 2]
```diff
@@ -58,4 +58,4 @@ def check_admin
     redirect_to authenticated_root_path, notice: 'Access Denied'
   end
 
-end
+end
```
Comment (ruba-ruba) [inline app/controllers/admin/questions_controller.rb]: changes like this should not be committed
- first of all file ending isn't correct
- if file does not have changed lines do not add it to commit

File: app/models/question.rb [hunk 3]
```diff
@@ -2,4 +2,6 @@ class Question < ActiveRecord::Base
   belongs_to :topic
   validates :content,  presence: true, uniqueness: { case_sensitive: false }
   validates :topic,    presence: true
+  has_many :questionstemplates
```
Comment (ruba-ruba) [inline app/models/question.rb]: `questionstemplates` => `questions_templates` => `question_templates`

File: app/models/questionstemplate.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+class Questionstemplate < ActiveRecord::Base
```
Comment (ruba-ruba) [inline app/models/questionstemplate.rb]: QuestionTemplate

**Pull request closed without merging**